docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -789,10 +810,10 @@ implementation("io.quarkus:quarkus-reactive-pg-client")
 quarkus.datasource.reactive.url=postgresql://localhost:5432/quarkus_test
 ----
 
-And you are ready to go.
+You are done.
```

Comment:
Reads unusual, maybe:

```suggestion
Now the tokens will be stored in the database
```

Reply:
Thanks for your suggestion, that is much better and provides more information. One small adjustment because  having "Now" and "will be" in the same sentence is slightly contradictory. What about:
"Now, the tokens are ready to be stored in the database."  
The next line 815 then provides more information: 
"By default, a database table used for storing tokens is created for you, however, you can disable this option with the `quarkus.oidc.db-token-state-manager.create-database-table-if-not-exists` configuration property."

@sberyozkin, would this work?

Reply:
Thanks for your thumbs up @sberyozkin. Updated per suggestion.